### PR TITLE
Support multiple db settings

### PR DESCRIPTION
**Pull request by ppworks**
If Application has multiple database settings, we need to keep other settings not only development.

### CODE REVIEW

File: lib/squasher.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def clean
 
   def rake(command, description = nil)
     tell(description) if description
-    system("bundle exec rake #{ command }")
+    system("bundle exec rake #{ command } 2>/dev/null")
```
Comment (ppworks) [inline lib/squasher.rb]: In some environment, we sometimes use same databases in spit of using multiple datatabases.

If there are duplicated database names in config/database.yml, an exception will be raised;"unkown database error" after drop 1st database or 
"database already exists" before create 2nd database.

So, drop these exceptions.

File: lib/squasher/config.rb [hunk 2]
```diff
@@ -68,18 +68,14 @@ def dbconfig
       return @dbconfig = nil unless File.exists?(dbconfig_file)
 
       begin
-        content = ERB.new(File.read(dbconfig_file)).result(binding)
-        @dbconfig = YAML.load(content)['development']
+        content = File.read(dbconfig_file).gsub(/database: (.+)/, 'database: squasher')
```
Comment (ppworks) [inline lib/squasher/config.rb]: Convert all database names.

**Comment by jalkoby** [comment]
@ppworks thanks for your contribution. I'd like to add it, but can you fix the Travis Ci issue before?

**Comment by ppworks** [comment]
Oops! Is this error?
<a href="https://travis-ci.org/jalkoby/squasher/jobs/103820309"></a>

I think it's bundler's issue.
Okay, I'll fix it.

See also
https://github.com/bundler/bundler/commit/f4481a7e3dbaf7f563e32afbe2cbbb84e3091f16

File: .travis.yml [hunk 3]
```diff
@@ -1,3 +1,5 @@
+before_install:
+  - gem update bundler
```
Comment (ppworks) [inline .travis.yml]: If bundler is too old in travis, some issues might be occurred. To avoid trouble, update bundler before install.

**Comment by jalkoby** [comment]
thanks for your contribution, @ppworks

**Comment by ppworks** [comment]
Thanks! I'm looking forward to bumping up verion.

**Comment by jalkoby** [comment]
sure! I'll make the changelog (it's time) and rise a patch version

**Pull request merged**